packages/story-editor/src/components/floatingMenu/elements/borderRadius.js

```diff
@@ -17,21 +17,63 @@
 /**
  * External dependencies
  */
-import { Icons } from '@googleforcreators/design-system';
+import { Icons, NumericInput } from '@googleforcreators/design-system';
 import { __ } from '@googleforcreators/i18n';
+import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
-import { IconButton, Separator } from './shared';
+import { useStory } from '../../../app';
+import { canSupportMultiBorder } from '../../../masks';
+import { DEFAULT_BORDER_RADIUS } from '../../panels/design/sizePosition/radius';
+import { Separator, useProperties } from './shared';
+
+const StyledInput = styled(NumericInput)`
+  width: 70px;
+  flex: 0 0 70px;
```

Comment:
I actually don't think both of these are needed. But the first one definitely was, otherwise the menu rendered all weird!